apis/pkg/v1/interfaces.go

```diff
@@ -542,21 +534,6 @@ func (p *ProviderRevision) SetSkipDependencyResolution(b *bool) {
 	p.Spec.SkipDependencyResolution = b
 }
 
-// GetWebhookTLSSecretName of this ProviderRevision.
-func (p *ProviderRevision) GetWebhookTLSSecretName() *string {
-	return p.Spec.WebhookTLSSecretName
```

Comment:
Any reason not to clean those out from the spec? https://github.com/crossplane/crossplane/blob/2e6a7277e315c4150332e12f9cd2e6c8cd421ede/apis/pkg/v1/revision_types.go#L91-L103

Reply:
Not really, I didn't want to remove fields from the API, but if we are fine with it we can definitely do that!

Reply:
I can take care of that in https://github.com/crossplane/crossplane/issues/4699
Just wanted to know if there was a reason.